Resources/Prototypes/_Mono/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/projectiles.yml

```
@@ -64,12 +64,56 @@
+    intensitySlope: 6
     totalIntensity: 1200
     maxTileBreak: 50
   - type: GatheringProjectile
   - type: MiningGatheringSoft
   - type: MiningGatheringHard
 
+- type: entity
+  id: ShipMissileASM302
+  name: ASM-302 missile
+  parent: BaseBulletTrigger
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)